ReferenceError: results is not defined
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\service\PhotoService.js:742:58)
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:177:8)
    at Promise.EventEmitter.emit (events.js:98:17)
    at Promise.emit (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:84:38)
    at Promise.fulfill (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:97:20)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\lib\model.js:1735:13
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\collection\aggregation.js:107:14
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1890:9
    at Server.Base._callHandler (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\base.js:448:41)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:481:18ReferenceError: results is not defined
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\service\PhotoService.js:785:57)
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:177:8)
    at Promise.EventEmitter.emit (events.js:98:17)
    at Promise.emit (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:84:38)
    at Promise.fulfill (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:97:20)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\lib\model.js:1735:13
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\collection\aggregation.js:107:14
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1890:9
    at Server.Base._callHandler (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\base.js:448:41)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:481:18ReferenceError: results is not defined
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\service\PhotoService.js:785:57)
    at Promise.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:177:8)
    at Promise.EventEmitter.emit (events.js:98:17)
    at Promise.emit (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:84:38)
    at Promise.fulfill (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mpromise\lib\promise.js:97:20)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\lib\model.js:1735:13
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\collection\aggregation.js:107:14
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\db.js:1890:9
    at Server.Base._callHandler (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\base.js:448:41)
    at E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:481:18Error: listen EADDRINUSE
    at errnoException (net.js:901:11)
    at HTTPServer.Server._listen2 (net.js:1039:14)
    at listen (net.js:1061:10)
    at HTTPServer.Server.listen (net.js:1127:5)
    at Object.<anonymous> (E:\src\TalkTakeSD\TalkTake\app.js:55:5)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)Error: failed to connect to [127.0.0.1:27017]
    at null.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:549:74)
    at EventEmitter.emit (events.js:106:17)
    at null.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\connection_pool.js:150:15)
    at EventEmitter.emit (events.js:98:17)
    at Socket.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\connection.js:533:10)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:441:14
    at process._tickCallback (node.js:415:13)Error: Redis connection to 127.0.0.1:6379 failed - read ECONNRESET
    at RedisClient.on_error (E:\src\TalkTakeSD\TalkTake\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\redis\index.js:106:14)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:441:14
    at process._tickCallback (node.js:415:13)Error: failed to connect to [127.0.0.1:27017]
    at null.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\server.js:549:74)
    at EventEmitter.emit (events.js:106:17)
    at null.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\connection_pool.js:150:15)
    at EventEmitter.emit (events.js:98:17)
    at Socket.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\mongoose\node_modules\mongodb\lib\mongodb\connection\connection.js:533:10)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:441:14
    at process._tickCallback (node.js:415:13)Error: Redis connection to 127.0.0.1:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (E:\src\TalkTakeSD\TalkTake\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (E:\src\TalkTakeSD\TalkTake\node_modules\redis\index.js:106:14)
    at Socket.EventEmitter.emit (events.js:95:17)
    at net.js:441:14
    at process._tickCallback (node.js:415:13)